documentation/modules/upgrading/con-upgrade-paths.adoc

```diff
@@ -6,31 +6,33 @@
 = Strimzi upgrade paths
 
 [role="_abstract"]
-Two Strimzi upgrade paths are possible.
+Two upgrade paths are available for Strimzi.
 
 Incremental upgrade::
-Upgrading Strimzi from the previous minor version to version {ProductVersion}.
+An incremental upgrade involves upgrading Strimzi from the previous minor version to version {ProductVersion}.
 
 Multi-version upgrade::
-Upgrading Strimzi from an old version to version {ProductVersion} within a single upgrade (skipping one or more intermediate versions).
-+
-For example, upgrading from Strimzi 0.24.0 directly to Strimzi {ProductVersion}.
-+
-NOTE: A multi-version Strimzi upgrade might still support the current version of a Kafka deployment.
+A multi-version upgrade involves upgrading an older version of Strimzi to version {ProductVersion} within a single upgrade, skipping one or more intermediate versions. 
+For example, upgrading directly from Strimzi 0.24.0 to Strimzi {ProductVersion} is possible. 
 
 [id='con-upgrade-paths-kafka-versions-{context}']
-== Supported Kafka versions
+== Support for Kafka versions when upgrading
+
+When upgrading Strimzi, it is important to ensure compatibility with the Kafka version being used.
+
+Only use a Kafka version that is supported by your version of Strimzi when upgrading. 
+You can upgrade to a higher supported Kafka version, and in some cases, you can also downgrade to an earlier supported Kafka version.
```

Comment:
This is confusing because when upgrading with Kafka version jump, you actually have to _use other Kafka version_. I would remove this paragraph.